spec/factories/kudos.rb

```diff
@@ -4,7 +4,8 @@
   factory :kudo do
     title { 'Title' }
     content { 'Content' }
-    giver_of_kudo
-    receiver_of_kudo
+    giver_of_kudo {}
+    receiver_of_kudo {}
```

Comment:
imo this naming is a bit off, because the giver and the receiver belong to the kudo anyway. "Maslo maslane" :)